waspc/data/Generator/templates/server/src/routes/operations/index.js

```diff
@@ -11,12 +11,8 @@ import {= importIdentifier =} from '{= importPath =}'
 
 const router = express.Router()
 
-{=# isAuthEnabled =}
-router.use(auth)
-{=/ isAuthEnabled =}
-
 {=# operationRoutes =}
-router.post('{= routePath =}', {= importIdentifier =})
+router.post('{= routePath =}', {=# isAuthEnabled =} auth, {=/ isAuthEnabled =} {= importIdentifier =})
```

Comment:
Never really worked with ExpressJS routers but a basic readup showed that adding the `auth` as an argument to `post/get` and before the callback should also install it as middleware.